obspy/core/util/testing.py

```diff
@@ -192,7 +283,7 @@ def __init__(self, image_path, image_name, reltol=1, *args, **kwargs):
         self.keep_only_failed = "OBSPY_KEEP_ONLY_FAILED_IMAGES" in os.environ
         self.output_path = os.path.join(image_path, "testrun")
         self.diff_filename = "-failed-diff.".join(self.name.rsplit(".", 1))
-        self.tol = get_matplotlib_defaul_tolerance() * reltol
+        self.tol = reltol * 3.0
```

Comment:
An RMSE of 3.0 essentially enables it to run across 1.3.x and 1.4.x as far as I can tell. For matplotlib versions smaller then 1.3, the tolerance is a lot higher. This assures the images are correct for newer matplotlib versions and for older ones it tests that the images get created and are not outrageously different.

I think this strike a good balance between test coverage and maintainability.